### PR TITLE
Fix Dockerfile

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

File: jupyter-covid19/Dockerfile [hunk 1]
```diff
@@ -5,15 +5,15 @@ USER root
 # COPY premade notebook
 RUN mkdir /home/jovyan/covid19-notebook
 
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/chore/splashPage/covid19-notebooks/splashPage.html /home/jovyan/covid19-notebook/
+ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/welcome.html /home/jovyan/covid19-notebook/
```
Comment (paulineribeyre) [inline jupyter-covid19/Dockerfile:8]:
```suggestion
ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/1.2.0/covid19-notebooks/welcome.html /home/jovyan/covid19-notebook/
```

File: jupyter-covid19/Dockerfile [hunk 2]
```diff
@@ -5,15 +5,15 @@ USER root
...
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/kaggle-demographics/kaggle_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.md /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/requirements.txt /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/covid19_seir.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/chicago-seir-forecast/seir_diagram.png /home/jovyan/covid19-notebook/
 RUN chmod -R 777 /home/jovyan/covid19-notebook
-RUN touch /home/jovyan/covid19-notebook/splashPage.html
+RUN touch /home/jovyan/covid19-notebook/welcome.html
```
Comment (paulineribeyre) [inline jupyter-covid19/Dockerfile:16]:
```suggestion
RUN touch /home/jovyan/welcome.html
```

Comment (atharvar28) [reply jupyter-covid19/Dockerfile:16]: `welcome.html` is added to the /home/jovyan/covid19-notebook/ folder in ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/1.2.0/covid19-notebooks/welcome.html /home/jovyan/covid19-notebook/`.

Comment (paulineribeyre) [reply jupyter-covid19/Dockerfile:16]: right, need to change both

**Pull request merged**